docs/install/install-edex.md

```diff
@@ -16,32 +16,20 @@ EDEX is the **E**nvironmental **D**ata **Ex**change system that represents the b
 - A **Solid State Drive (SSD)** is recommended
     - A SSD should be mounted either to `/awips2` (to contain the entire EDEX system) or to `/awips2/edex/data/hdf5` (to contain the large files in the decoded data store). EDEX can scale to any system by adjusting the incoming LDM data feeds or adjusting the resources (CPU threads) allocated to each data type.
 
-> Note: EDEX is only supported for 64-bit CentOS and RHEL 7 Operation Systems.  You may have luck with Fedora Core 12 to 14 and Scientific Linux.
+> Note: EDEX is only supported for 64-bit CentOS and RHEL 7 Operating Systems.
```

Comment:
Maybe we could change this to the "new style" note as well.

`!!! note "EDEX is only supported for 64-bit CentOS and RHEL 7 Operating Systems."`